### PR TITLE
docs(http): Add missing desc for uploading a file

### PR DESCRIPTION
**Pull request by albohlabs**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [] Tests for the changes have been added (for bug fixes / features)
- [] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[] Documentation content changes
[ ] angular.io application / infrastructure changes
[x ] Other... Please describe: Add documentation for file upload via httpclient
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by albohlabs** [comment]
I signed it!

File: aio/content/guide/http.md [hunk 1]
```diff
@@ -254,6 +254,17 @@ http
 
 In this way, you send the POST request to the URL `/api/items/add?id=3`.
 
+### Upload a file
+
+Another common task is to upload a file to the server. You can archive this by using a `FormData`:
```
Comment (alfaproject) [inline aio/content/guide/http.md]: `achieve`

Comment (albohlabs) [reply aio/content/guide/http.md]: Thanks. Changed. 😅

**Comment by rafaelss95** [comment]
Isn’t it already being covered by #20920?

**Comment by albohlabs** [comment]
You are right. Thanks for the hint.

**Pull request closed without merging**